layers/+lang/lean/packages.el

```diff
@@ -0,0 +1,34 @@
+(setq dotspacemacs-additional-packages
+      (append '(dash dash-functional f s
+                     fill-column-indicator
+                     mmm-mode)))
```

Comment:
Thank you for helping me @person808, I'm almost done. However, I think I have a valid reason for adding things in `dotspacemacs-additional-packages`. The packages are "utility libraries," such as `s` which is a string manipulation library. For this reason, I don't think any package in particular should "own" these sorts of libraries.